library/Ivoz/Kam/Domain/Service/Dispatcher/UpdateByApplicationServer.php

```diff
@@ -49,7 +49,7 @@ public function execute(ApplicationServerInterface $entity)
             ->setApplicationServerId($entity->getId())
             ->setSetid(1)
             ->setDestination('sip:' . $entity->getIp() . ":6060")
-            ->setDescription($entity->getName());
+            ->setDescription($entity->getName() ?? '');
```

Comment:
We should create another issue for fixing this: Applications Server name should not be nullable.